wiki/markdown/configuration.md

```diff
@@ -28,4 +28,17 @@ This situation may be needed when the events throughput is very high. In this ca
   <img src="images/multiple_EI_instances_same_rule.png">
 </img>
   
+### Confiure Eiffel-Intelligence and Extraction Rules for specific Eiffel protocol versions
```

Comment:
Confiure  -> Configure. Also, the header should maybe be "Configure Eiffel Intelligence with extraction rules", with instead of and. Version in singular (since only one version of the protocol can be used at the same time)

Reply:
Fixed.